homeassistant/components/faadelays/strings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "title": "FAA Delays",
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "id": "Airport"
```

Comment:
Maybe mention a format one needs to put in?

Reply:
Added a title and description to the user step to clarify the IATA formatting requirement. Thanks!